### PR TITLE
VPA - Optional RenewTime fix

### PR DESCRIPTION
**Pull request by krzysied**
RenewTime is optional Lease field. Requires not-nil check.

### CODE REVIEW

File: vertical-pod-autoscaler/pkg/utils/status/status_object.go [hunk 1]
```diff
@@ -124,7 +124,8 @@ func IsStatusValid(status *apicoordinationv1.Lease, leaseTimeout time.Duration)
 
 func isStatusValid(status *apicoordinationv1.Lease, leaseTimeout time.Duration, now time.Time) bool {
 	return status.CreationTimestamp.Add(leaseTimeout).After(now) ||
```
Comment (bskiba) [inline vertical-pod-autoscaler/pkg/utils/status/status_object.go]: CreationTimestamp is not optional I presume?

File: vertical-pod-autoscaler/pkg/utils/status/status_object_test.go [hunk 2]
```diff
@@ -230,6 +230,16 @@ func TestIsStatusValid(t *testing.T) {
 			leaseTimeout:  10 * time.Second,
 			expectedValid: true,
 		},
+		{
+			name: "Outdated CreationTimestamp",
```
Comment (bskiba) [inline vertical-pod-autoscaler/pkg/utils/status/status_object_test.go]: Useful to point out this actually tests nil RenewTime

Comment (bskiba) [reply vertical-pod-autoscaler/pkg/utils/status/status_object_test.go]: (Could be just by making RenewTime = nil explicit in the lease object or changing test case name to Outdated CreationTimestamp with no/nil RenewTime)

File: vertical-pod-autoscaler/pkg/utils/status/status_object.go (hunk 1 above)
Comment (krzysied) [reply vertical-pod-autoscaler/pkg/utils/status/status_object.go]: AFIAK it is not.

File: vertical-pod-autoscaler/pkg/utils/status/status_object_test.go (hunk 2 above)
Comment (krzysied) [reply vertical-pod-autoscaler/pkg/utils/status/status_object_test.go]: Test case name updated.

**Pull request merged**